server/auth_test.go

```diff
@@ -273,3 +275,46 @@ func TestNoAuthUser(t *testing.T) {
 		})
 	}
 }
+
+func TestNoAuthUserNoConnectProto(t *testing.T) {
+	conf := createConfFile(t, []byte(`
+		listen: "127.0.0.1:-1"
+		accounts {
+			A { users [{user: "foo", password: "pwd"}] }
+		}
+		authorization { timeout: 1s }
```

Comment:
Actually, duration is not supported here, so it has to be a float (or simply 1).

Reply:
Did not complain, and properly set it to 1s vs 2s. So I think it worked.

Reply:
It does not, just tried with `10s` in a manual test and got kicked out quickly. Probably defaulted to 1 sec

Reply:
https://github.com/nats-io/nats-server/blob/7aba8a8e9ee7f44302c1965d61e11e2491c5a43c/server/opts.go#L3220

Reply:
Ah, I see.